javascript-modal/main.js

```diff
@@ -0,0 +1,16 @@
+const popup = document.getElementById('myPopup');
+const btn = document.getElementById('mybtn');
+const span = document.getElementsByClassName('close')[0];
+
+btn.onclick = function () {
+  popup.style.display = 'block';
```

Comment:
We want to avoid inline styling as much as possible. We want to be utilizing classes to apply / remove styling